lib/course-db.js

```diff
@@ -3,6 +3,7 @@ var path = require('path');
 var _ = require('lodash');
 var fs = require('fs');
 var async = require('async');
+var moment = require('moment-timezone');
```

Comment:
Why not just use plain `moment` here? I don't think we need the timezone capabilities?

Reply:
*shrug* Habit.